Code/GraphMol/FileParsers/CMakeLists.txt

```diff
@@ -40,20 +40,24 @@ rdkit_test(fileParsersTest1 test1.cpp
            LINK_LIBRARIES FileParsers SmilesParse
 	   SubstructMatch GraphMol RDGeneral RDGeometryLib ${maeparser_var})
 
-if(RDK_BUILD_TEST_GZIP)
+
```

Comment:
there's some duplication here. I think you meant to delete this block

Reply:
Will do.